test/local/account_routes.js

```diff
@@ -6,7 +6,7 @@ require('ass')
 
 var sinon = require('sinon')
 
-var test = require('../ptaptest')
+var test = require('tap').test
```

Comment:
Yeap, that feature was added in https://github.com/tapjs/node-tap/commit/4aa1086a87b31f78b36ae2369ced98bdf34691a3